src/Microsoft.Health.CosmosDb/Configs/CosmosDataStoreConfiguration.cs

```diff
@@ -20,6 +20,12 @@ public class CosmosDataStoreConfiguration
 
         public int? InitialDatabaseThroughput { get; set; }
 
+        public int? InitialFhirCollectionThroughput { get; set; }
```

Comment:
The `InitialFhirCollectionThroughput`, `ControlPlaneCollectionId`, and `InitialControlPlaneCollectionThroughput` aren't needed either.

Reply:
Done